js/w3c/headers.js

```diff
@@ -155,6 +156,18 @@ define(
                     ret += ", <span class='ed_mailto'><a class='u-email email' " + rm + " href='mailto:" + p.mailto + "'>" + p.mailto + "</a></span>";
                 }
                 if (p.note) ret += " (" + p.note + ")";
+                if (p.extras && p.extras.length !== 0) {
```

Comment:
@tobie, @halindrome, as other maintainers, do you have an opinion? Would it be ok for me to start switching to using a more functional style?